eng/deploy.yaml

```diff
@@ -0,0 +1,218 @@
+        ScriptArguments: -RuleName UnblockSQLForUpgrade -Add -ConnectionString "$(maestro-admin-sql-connection-string)"
+        azurePowerShellVersion: LatestVersion
+    - task: PowerShell@2
+      displayName: update-database.ps1
+      inputs:
+        targetType: filePath
+        filePath: $(Pipeline.Workspace)/Maestro.Data/update-database.ps1
+        arguments: Maestro.Data.dll
+        workingDirectory: $(Pipeline.Workspace)/Maestro.Data
+      env:
+        BUILD_ASSET_REGISTRY_DB_CONNECTION_STRING: $(maestro-admin-sql-connection-string)
+    - task: AzurePowerShell@3
+      displayName: eng\firewall.ps1 -Remove
+      inputs:
+        azureSubscription: ${{ parameters.Subscription }}
+        ScriptPath: $(Pipeline.Workspace)/ReleaseUtilities/firewall.ps1
+        ScriptArguments: -RuleName UnblockSQLForUpgrade -Remove -ConnectionString "$(maestro-admin-sql-connection-string)"
+        azurePowerShellVersion: LatestVersion
+      condition: always()
+
+  - job: deployMaestro
+    displayName: Deploy maestro service fabric application
+    dependsOn: 
+    - updateDatabase
+    - approval
+    steps:
+    - download: current
+      artifact: MaestroApplication
+    - task: ServiceFabricDeploy@1
+      displayName: Deploy Service Fabric Application (Maestro)
+      inputs:
+        applicationPackagePath: $(Pipeline.Workspace)/MaestroApplication/applicationpackage
+        serviceConnectionName: ${{ parameters.ServiceFabricConnection }}
+        publishProfilePath: $(Pipeline.Workspace)/MaestroApplication/projectartifacts/PublishProfiles/$(PublishProfile).xml
+        applicationParameterPath: $(Pipeline.Workspace)/MaestroApplication/projectartifacts/ApplicationParameters/$(PublishProfile).xml
+
+  - job: deployTelemetry
+    displayName: Deploy telemetry service fabric application
+    dependsOn: 
+    - approval
+    steps:
+    - download: current
+      artifact: TelemetryApplication
+    - task: ServiceFabricDeploy@1
+      displayName: Deploy Service Fabric Application (Telemetry)
+      inputs:
+        applicationPackagePath: $(Pipeline.Workspace)/TelemetryApplication/applicationpackage
+        serviceConnectionName: ${{ parameters.ServiceFabricConnection }}
+        publishProfilePath: $(Pipeline.Workspace)/TelemetryApplication/projectartifacts/PublishProfiles/$(PublishProfile).xml
+        applicationParameterPath: $(Pipeline.Workspace)/TelemetryApplication/projectartifacts/ApplicationParameters/$(PublishProfile).xml
+
+  - job: deployStatus
+    displayName: Deploy dotnet-status web app
+    dependsOn: 
+    - approval
+    steps:
+    - download: current
+      artifact: DotNetStatus
+    - task: AzureRmWebAppDeployment@4
+      inputs:
+        ConnectionType: AzureRM
+        azureSubscription: ${{ parameters.Subscription }}
+        appType: webApp
+        WebAppName: ${{ parameters.DotNetStatusAppName }}
+        deployToSlotOrASE: true
+        ResourceGroupName: monitoring
+        SlotName: staging
```

Comment:
Should the be production for Prod?

Reply:
I think this is the web app service staging slot, I think staging is still right, but we didn't add a staging slot to that web service. I believe @ChadNedzlek added one and I'mretrying that stage now

Reply:
This staging means what staging is really supposed to mean. It's the place you "stage" something you are rolling out. Once it's "staged", you swap it into production in a seamless way.

The "our team" use of the word staging is what most services would call "int" or "pre-production".  Which is definitely super confusing. :-)